### PR TITLE
Portfolio: Correct accessibility issues

### PR DESCRIPTION
**Pull request by SamTush**
In this branch, I correct 25 accessibility issues identified by axe DevTools.

- correct contrast of logo in the nav.
- correct contrast of the contact-form background.
- put better alt for all images.
- Change the ID name of elements that had the same ID name.

1. [ ] Page titles - change color for better contrast.
2. [ ] Image text alternatives** - put better alt text
3. [ ] Text headings - change color for better contrast.
4. [ ] Color contrast - change the contact-form background color.
5. [ ] Resize - no issues found.
6. [ ] Interaction - give contract-form labels
7. [ ] Moving content - no issues found
8. [ ] Multimedia - no issues found
9. [ ] The basic structure of the page - no issues found

Screenshot 2023-02-01 at 18 28 34
Screenshot 2023-02-01 at 18 28 18

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h4>CANOPY</h4>
                     <button><a>See Project</a></button>
                 </div>
             </div>
-            <div  id="reverse" class="grid-item grid-item-2 row-reverse">
+            <div  id="reverse-one" class="grid-item grid-item-2 row-reverse">
```
Comment (SamTush) [inline index.html:89]: Change ID name

File: style.css [hunk 2]
```diff
@@ -97,7 +97,7 @@ ul {
   font-weight: 600;
   font-size: 18px;
   line-height: 20px;
-  color: #6070ff;
+  color: #2233CA;
```
Comment (SamTush) [inline style.css]: Change color to something darker.

**Review by SamTush — COMMENTED** [review]
lines of code I change to improve on the web accessibility.

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @SamTush , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] Great job your page is accessible at 98% but you can just improve a bit the performance :+1: 
 
image


- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
-  [ ] [OPTIONAL] Kindly  Add a meaningful description in your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

**Pull request merged**